### PR TITLE
Log Level GA (7/10)

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Beta --> GA - Log level detection feature

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204613338792107

### CODE REVIEW

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, since beta is complete, [log-level](https://github.com/SumoLogic/sumologic-documentation/pull/2704/files#diff-3eaef3edc9480eee9ed9d32ac7bcb0d3bcc76892bd85bf38905ed4ca528e270b) file needs to be added to sidebars.ts.

**Comment by kimsauce** [comment]
> @kimsauce, since beta is complete, [log-level](https://github.com/SumoLogic/sumologic-documentation/pull/2704/files#diff-3eaef3edc9480eee9ed9d32ac7bcb0d3bcc76892bd85bf38905ed4ca528e270b) needs to be added to sidebars.ts.

great catch!

File: sidebars.ts [hunk 1]
```diff
@@ -2450,12 +2451,12 @@ integrations: [
           items: [
             'cse/automation-service/about-automation-service',
             'cse/automation-service/automation-service-automations',
-            'cse/automation-service/automation-service-app-central',  
+            'cse/automation-service/automation-service-app-central',
```
Comment (kimsauce) [inline sidebars.ts:2454]:
```suggestion
            'cse/automation-service/automation-service-app-central', 
```

File: sidebars.ts [hunk 2]
```diff
@@ -2450,12 +2451,12 @@ integrations: [
...
```
Comment (kimsauce) [inline sidebars.ts:2454]:
```suggestion
            'cse/automation-service/automation-service-app-central',
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce. Approved.

**Comment by jpipkin1** [comment]
@kimsauce, it looks like this PR is a duplicate of PR #2567.

**Comment by kimsauce** [comment]
> @kimsauce, it looks like this PR is a duplicate of PR #2567.

Oh yeah, I saw that this morning! Just merged the two together.

File: blog-service/2023-07-14-log-search.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: July 14, 2023 (Search)
```
Comment (tarunk2) [inline blog-service/2023-07-14-log-search.md:2]:
```suggestion
title: July 10, 2023 (Search)
```

**Review by tarunk2 — APPROVED** [review]

**Pull request merged**